pkg/server/v3/delta_test.go

```diff
@@ -457,3 +458,95 @@ func TestDeltaCallbackError(t *testing.T) {
 		})
 	}
 }
+
+// BENCHMARKS =====================================================================================================
+
+func BenchmarkDeltaResponseHandlers(b *testing.B) {
+	for _, typ := range testTypes {
+		b.Run(typ, func(b *testing.B) {
+			for n := 0; n < b.N; n++ {
+				config := makeMockConfigWatcher()
+				config.deltaResponses = makeDeltaResponses()
+				s := server.NewServer(context.Background(), config, server.CallbackFuncs{})
+
+				resp := makeMockDeltaStream(&testing.T{})
```

Comment:
same here, i think this will ignore all the failures?

Reply:
I think you're right, how should we go about that? We know this will work which is why I just dumped that in there, but we'd have to refactor that to support benchmarking if we wanted to make that big of a change. Unless I'm missing a trivial way of addressing this

Reply:
define an interface `type Errors interface { Error(string), Errorf(string) }` so that it can accept either a T or a B